pyvcloud/vcd/org.py

```diff
@@ -638,3 +642,134 @@ def update_org(self, org_name, is_enabled=None):
                                                 org_admin_resource,
                                                 EntityType.ADMIN_ORG.value)
         return org_admin_resource
+
+    def create_org_vdc(self,
+                       vdc_name,
+                       provider_vdc_name,
+                       description='',
+                       allocation_model='AllocationVApp',
+                       cpu_units='MHz',
+                       cpu_allocated=0,
+                       cpu_limit=0,
+                       mem_units='MB',
+                       mem_allocated=0,
+                       mem_limit=0,
+                       nic_quota=0,
+                       network_quota=0,
+                       vm_quota=0,
+                       storage_profiles=[],
+                       resource_guaranteed_memory=None,
+                       resource_guaranteed_cpu=None,
+                       vcpu_in_mhz=None,
+                       is_thin_provision=None,
+                       network_pool_name=None,
+                       uses_fast_provisioning=None,
```

Comment:
i think is_fast_provisioning_enabled or is_fast_provisioning is a better name than uses_fast_provisioning

Reply:
I think so to, but in this case we should map the names from the vCD API schema as close as possible: http://pubs.vmware.com/vcd-55/topic/com.vmware.vcloud.api.reference.doc_55/doc/types/CreateVdcParamsType.html